src/server-extension/resolvers/StateResolver/index.ts

```diff
@@ -73,4 +73,63 @@ export class StateResolver {
   processorState(@Root() state: ProcessorState): ProcessorState {
     return state
   }
+
+  @Query(() => EarningStatsOutput)
+  async totalJoystreamEarnings(): Promise<EarningStatsOutput> {
+    const em = await this.tx()
+    const result = (
+      await em.query<
+        {
+          total_rewards_volume: string
+          total_crt_volume: string
+          total_nft_volume: string
+        }[]
+      >(
+        `
+      SELECT
+          SUM(
+          COALESCE(event.data->>'amount', '0')::bigint
+        ) AS "total_rewards_volume",
+        SUM(
+          COALESCE(amm_buy.price_paid, '0')::bigint
+        ) AS "total_crt_volume",
+        SUM(
+          COALESCE(event.data->>'price', '0')::bigint + COALESCE(winning_bid.amount, 0)
+        ) AS "total_nft_volume"
+      FROM
+          "event"
+      LEFT JOIN amm_transaction AS amm_buy ON "data"->>'ammMintTransaction' = amm_buy.id
+      LEFT JOIN bid AS winning_bid ON "data"->>'winningBid' = winning_bid.id
+      WHERE
+          "event"."data"->>'isTypeOf' IN (
+              'ChannelPaymentMadeEventData',
```

Comment:
```suggestion
              'ChannelPaymentMadeEventData',
              'ChannelRewardClaimedEventData',
              'ChannelRewardClaimedAndWithdrawnEventData',
```

I think you should have to more event types for total rewards volume